GitCommands/Git/GitCommandHelpers.cs

```diff
@@ -54,6 +54,19 @@ public enum IgnoreSubmodulesMode
         All = 5
     }
 
+    /// <summary>Mode for 'git clean'</summary>
+    public enum CleanMode
+    {
+        /// <summary>Only untracked files not in .gitignore, the default</summary>
+        OnlyNonIgnored = 1,
```

Comment:
```suggestion
        OnlyNonIgnored = 0,
```
Make it the default.

Reply:
All other extensions start with (including those with defaults), so I suggest keeping 1.

Reply:
Changed for all existing enums too